clients/drcachesim/scheduler/scheduler.cpp

```diff
@@ -3404,6 +3426,7 @@ scheduler_tmpl_t<RecordType, ReaderType>::next_record(output_ordinal_t output,
                     preempt = !need_new_input;
                     need_new_input = true;
                     input->instrs_in_quantum = 0;
+                    ++outputs_[output].stats[memtrace_stream_t::SCHED_STAT_TIME_PREEMPTS];
```

Comment:
It seems a bit confusing for instruction-based preempts to be counted as _time_ preempts.  Maybe remove "TIME" from the stat name and just call it SCHED_STAT_PREEMPTS or possibly SCHED_STAT_QUANTUM_PREEMPTS?

Reply:
There are priority preempts also which this is not, so QUANTUM is better.